js/script.js

```diff
@@ -884,10 +889,10 @@ function commandMessage(key) {
             "Install Nuget package&nbsp;<a href='https://www.nuget.org/packages/Microsoft.AI.MachineLearning' target='_blank'>Microsoft.AI.MachineLearning</a>",
 
         "windows,Java,X64,DefaultCPU":
-            "Follow <a href='https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#common-build-instructions' target='_blank'>build</a> and <a href='https://aka.ms/onnxruntime-java' target='_blank'>API instructions</a>",
+            "Add dependency on <a href='https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime' target='_blank'>com.microsoft.onnxruntime:onnxruntime</a> using Maven",
```

Comment:
This could say "Maven/Gradle" as both will accept a reference to a maven central repo.

Reply:
thanks, updated!